bot/cogs/reminders.py

```diff
@@ -56,12 +56,21 @@ def __init__(self, bot: Bot):
                 self.schedule_task(loop, reminder["id"], reminder)
 
     @staticmethod
-    async def _send_confirmation(ctx: Context, on_success: str) -> None:
+    async def _send_confirmation(
+        ctx: Context, on_success: str, reminder_id: str, delivery_dt: Optional[datetime]
+    ) -> None:
         """Send an embed confirming the reminder change was made successfully."""
         embed = Embed()
         embed.colour = Colour.green()
         embed.title = random.choice(POSITIVE_REPLIES)
         embed.description = on_success
+
+        if delivery_dt:
+            embed.set_footer(text=f"ID: {reminder_id}, Due: {delivery_dt.strftime('%Y-%m-%dT%H:%M:%S')}")
+        else:
+            # Reminder deletion will have a `None` `delivery_dt`
+            embed.set_footer(text=f"ID: {reminder_id}")
```

Comment:
If a string is made with the default value being just the ID, the due date could be conditionally appended to it in order to cut down a bit on redundant code.